src/elements/HomePage.jsx

```diff
@@ -0,0 +1,5 @@
+import NavBar from "./Navbar"
+
+export default function HomePage(){
+  <NavBar />
+}
```

Comment:
Make sure you have an extra empty line (EOF line) for each file! This can be problematic in certain situations and generally just good practice.